MultiEditUtils.sublime-settings

```diff
@@ -1,3 +1,7 @@
 {
-  "live_split_selection" : true
+  "live_split_selection" : true,
+  // whether the tab key should jump to the next field during the selection field mode
+  "selection_fields_tab_enabled": true,
+  // whether the escape key should cancel the selection field mode
+  "selection_fields_escape_enabled": true

```

Comment:
I'm not really sure why these two options exist. If I were to set these two options to false, how would I cycle between the fields and escape this mode? In which scenario would I want to set these options to false?
From my current understanding these options could be removed or am I missing something?